packages/mdl-card/README.md

```diff
@@ -0,0 +1,110 @@
+# mdl-card
+
+MDL card is a component that implements the
+[Material Design card component](https://spec.googleplex.com/quantum/components/cards.html), and makes it available to
+developers as a set of CSS classes.
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+
+## Usage
+
+```html
+<div class="mdl-card demo-card">
```

Comment:
we may want to add in here that cards don't have a default height (e.g. the `demo-card` modifier on here) so if you want every card to be the same size you need to specify as such.